pkg/batch/executor.go

```diff
@@ -11,19 +11,31 @@ import (
 // dispatching new ones would start blocking.
 const RequestBufferSize = 1 << 17
 
+type Executable interface {
```

Comment:
I think this should be

```suggestion
type Executer interface {
```

according to https://golang.org/doc/effective_go#interface-names.